fcrepo4/fcrepo4
# Add root path to velocity context for static CSS and JS resources.

**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-2534

Related PR: https://github.com/fcrepo4/fcrepo4/pull/1208
Related external ticket: https://opensource.ncsa.illinois.edu/jira/browse/NDS-644

# What does this Pull Request do?
Determines the context of the application and pushes that into the Velocity context to be used in templates to allow referencing static resources directly.

# What's new?
Uses the `uriInfo`'s `getBaseUri` to determine application context, splits on `/` and strips the last element to get outside the Jersey context.

Moves the `common.css` and `common.js` into the `../webapp/static/(css|js)` directories with all the other files.

Removes the ViewAssets class that only was used to serve commons.css and commons.js.

Removes the static variables in StreamingBaseHtmlProvider with location of commons.css and commons.js

# How should this be tested?

1. This should build
2. The HTML UI should be functional when accessed via Jersey, Tomcat or any chosen deployment.
3. Specifically it is meant to alleviate forcing the scheme for these resources when being referenced through a proxy.

Example:
* Does this change require documentation to be updated? no
* Does this change add any new dependencies?  no
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)?  no
* Could this change impact execution of existing code? no

# Interested parties
@c1mckay / @craig-willis / @awoods
